sdk-setup/src/sdk-assistant

```diff
@@ -210,7 +212,7 @@ case ${1:-} in
         ;;
 esac
 
-[[ $# -gt 0 ]] || fail "command expected"
+[[ $# -gt 0 ]] || usage quit
```

Comment:
I would prefer to still say "command expected" or so in addition to dumping the help. We should also write to stderr and exit with non-zero here.